src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -429,3 +429,18 @@ module:io.netty.* -app
 module:newrelic -app -group
 path:**/*newrelic*/** -app -group
 path:**/*LogRocket*/** -app -group
+
+## Unreal Engine
+
+### Unreal Engine internal assertion handling
+family:native stack.function:FDebug::CheckVerifyFailedImpl* ^-app -app v+app ^-group -group v+group
+
+### Unreal Engine internal ensure handling
+family:native stack.function:UE::Assert::Private::ExecCheckImplInternal* ^-app -app v+app ^-group -group v+group
+
+### UE SDK USentrySubsystem event capturing
+family:native stack.function:USentrySubsystem::CaptureMessage* ^-app -app v+app ^-group -group v+group
+family:native stack.function:USentrySubsystem::CaptureMessageWithScope* ^-app -app v+app ^-group -group v+group
+family:native stack.function:USentrySubsystem::CaptureEvent* ^-app -app v+app ^-group -group v+group
+family:native stack.function:USentrySubsystem::CaptureEventWithScope* ^-app -app v+app ^-group -group v+group
+family:native stack.function:USentrySubsystem::*execCapture* ^-app -app v+app ^-group -group v+group
```

Comment:
[This](https://ivan-tustanivskyi.sentry.io/issues/6234614158/events/9780b099b6584920ad528557f56ef9dd/?project=4508533445754880) is an example of this rule:
<img width="909" alt="image" src="https://github.com/user-attachments/assets/b7075d9a-3ded-4e3e-b3e4-6a6824d6b0e5" />